config/config_test.go

```diff
@@ -54,11 +54,14 @@ func TestFlowConfig_Validate(t *testing.T) {
 	// set invalid config values
 	c.NetworkConfig.UnicastConfig.UnicastRateLimitersConfig.MessageRateLimit = -100
 	c.NetworkConfig.UnicastConfig.UnicastRateLimitersConfig.BandwidthRateLimit = -100
+	// InitDecayUpperBound should be greater than field InitDecayLowerBound
+	c.NetworkConfig.GossipSubConfig.GossipSubScoringRegistryConfig.InitDecayLowerBound = .9
+	c.NetworkConfig.GossipSubConfig.GossipSubScoringRegistryConfig.InitDecayUpperBound = .5
 	err := c.Validate()
 	require.Error(t, err)
 	errs, ok := errors.Unwrap(err).(validator.ValidationErrors)
 	require.True(t, ok)
-	require.Len(t, errs, 2)
+	require.Len(t, errs, 3)
```

Comment:
This only checks that there were 3 errors returned but it doesn't check what those errors were. Would be better to check the actual errors returned.